app/src/main/java/com/cody/androidexercise/ui/facebook/FacebookLoginViewModel.kt

```diff
@@ -5,6 +5,7 @@ import androidx.lifecycle.ViewModel
 import com.cody.androidexercise.credential.CredentialRepository
 import com.facebook.*
 import com.facebook.login.*
```

Comment:
i think no need for this line because you already have **import com.facebook.***